src/chains/chains.service.spec.ts

```diff
@@ -1,20 +1,43 @@
-import { Test, TestingModule } from '@nestjs/testing';
-import { ConfigServiceModule } from '../services/config-service/config-service.module';
+import { ConfigService } from '../services/config-service/config-service.service';
+import { TransactionServiceManager } from '../services/transaction-service/transaction-service.manager';
+import { TransactionService } from '../services/transaction-service/transaction-service.service';
 import { ChainsService } from './chains.service';
+import { Backbone } from './entities/backbone.entity';
```

Comment:
nitpick: This import can be shortened.

Reply:
Fixed in [857b5ab](https://github.com/5afe/safe-client-gateway-nest/pull/30/commits/857b5ab863510c87140f573ba4405527c1914975)